json/src/spec/authority_round.rs

```diff
@@ -49,6 +49,9 @@ pub struct AuthorityRoundParams {
 	/// Block from which monotonic steps start.
 	#[serde(rename="validateStepTransition")]
 	pub validate_step_transition: Option<Uint>,
+	/// Whether transitions should be immediate.
+	#[serde(rename="immediateTransitions")]
+	pub immediate_transitions: Option<bool>,
```

Comment:
Transitions to `SimpleList` in a `Multi` set are instant regardless of the setting of this flag. It could be a per-validator-set flag, but it makes more sense to have it global because
  a) this flag influences whether finality checking is done, which is at the engine level
  b) if any single validator set doesn't finalize transitions as necessary, then warp sync and light sync won't work and it's the same effect as having none of the sets finalize transitions.